bitgouel-api/src/main/kotlin/team/msg/thirdparty/feign/company/client/CompanyClient.kt

```diff
@@ -0,0 +1,13 @@
+package team.msg.thirdparty.feign.company.client
+
+import org.springframework.cloud.openfeign.FeignClient
+import org.springframework.web.bind.annotation.GetMapping
+import team.msg.domain.company.presentation.data.response.CompaniesResponse
+
+@FeignClient(value = "company", url = "http://localhost:8080")
+interface CompanyClient {
+
+    @GetMapping("/company")
+    fun queryCompanies(): CompaniesResponse
```

Comment:
```suggestion
@FeignClient(value = "company", url = "http://localhost:8080/company")
interface CompanyClient {

    @GetMapping
    fun queryCompanies(): CompaniesResponse
```

company만의 OpenFeign으로 보이는데 이렇게 쓰는 건 어떨까요??

Reply:
d96053fa0d6e08797e77accae0f78010c5f6e9af 좋네요. 적용했습니다 ~